src/main/java/org/betonquest/betonquest/quest/event/conversation/ConversationEventFactory.java

```diff
@@ -60,11 +60,37 @@ public Event parseEvent(final Instruction instruction) throws InstructionParseEx
         } catch (final ObjectNotFoundException e) {
             throw new InstructionParseException(e.getMessage(), e);
         }
+        final String startingOption = getStartOption(instruction, conversationID);
         return new PrimaryServerThreadEvent(
                 new OnlineProfileRequiredEvent(
-                        loggerFactory.create(ConversationEventFactory.class), new ConversationEvent(loggerFactory, conversationID), instruction.getPackage()
+                        loggerFactory.create(ConversationEventFactory.class), new ConversationEvent(loggerFactory, conversationID, startingOption), instruction.getPackage()
                 ), server, scheduler, plugin
         );
     }
+
+    /**
+     * Gets an optional start option for the conversation.
+     * This can't verify if the option is actually valid
+     * because the conversation data is not loaded at event creation.
```

Comment:
```suggestion
     * We need to manually verify the existence of the starting option
     * because the conversation data is not yet loaded at event creation.
```

Reply:
I think this should be an inline comment since it is about the implementation, not the behaviour.